src/opendlv-device-adc-bbblue.cpp

```diff
@@ -66,28 +66,28 @@ int32_t main(int32_t argc, char **argv) {
         voltageReading.voltage(voltage);
       
         opendlv::proxy::DistanceReading distanceReading;
-        // dIR0 = 12.174 * powf((d0) * (3.0 / 4096), -1.051);
-        float const distance{12.174f * powf(voltage, -1.051f)};
-        if(voltage >= 3.0f && distance <= 40) {
-          distanceReading.distance(distance/100.f);
+        float distance{(1.0f / (voltage / 10.13f)) - 3.8f };
+        if(voltage >= 3.0f && distance <= 40.0f) {
```

Comment:
I just noticed: floats should not be compared with ==

Reply:
I always assumed >= and <= were fine since it's simply adding a single value (either 3.0 or 40.0) to the range, plus the compiler didn't complain. 
However I can quickly change it if you prefer so.